### DISCUSSION TITLE
How to increase zoom level

### DISCUSSION THREAD

**Discussion by Prasanna-kumar-Patra** [google/model-viewer#4126] [Q&A]
How to increase zoom level in editor and viewer also not supporting after zooming z also

**Comment by elalish** [comment]
I don't quite follow - can you share some screenshots of what you've tried and what you're trying to accomplish?

**Comment by bigrig2212** [comment]
Same question here. Just trying to set the model so that it's a little zoomed out initially - so that it doesn't clip the bounding box when animating (see bottom of frame). Just can't seem to find a zoom control in all the docs..?

<img width="428" alt="Screenshot 2023-05-15 at 4 01 35 PM" src="https://github.com/google/model-viewer/assets/19869387/5aadbb5d-1d17-4b7b-9d19-b7be0151f499">

**Reply by bigrig2212** [reply]
i see - it's supposed to be settable on camera-orbit... but it doesn't work. the last param does't do anything. these two snippets render an identical model:

```                           
<model-viewer id="npc-demo" class="model-viewer-big" 
                                    camera-controls 
                                    touch-action="pan-y" 
                                    autoplay 
                                    animation-name="Running" 
                                    ar ar-modes="webxr scene-viewer" 
                                    shadow-intensity="1" 
                                    exposure="0.75"
                                    src="/assets/models_home/{{character_path}}" 
                                    camera-orbit="9.698deg 72.69deg 200m" 
                                    disable-tap
                                    alt="Npc demo">
```

```
                           <model-viewer id="npc-demo" class="model-viewer-big" 
                                    camera-controls 
                                    touch-action="pan-y" 
                                    autoplay 
                                    animation-name="Running" 
                                    ar ar-modes="webxr scene-viewer" 
                                    shadow-intensity="1" 
                                    exposure="0.75"
                                    src="/assets/models_home/{{character_path}}" 
                                    camera-orbit="9.698deg 72.69deg 2m" 
                                    disable-tap
                                    alt="Npc demo">
```

**Reply by elalish** [reply]
If you want to set the `camera-orbit` outside of it's usual bounds, you also need to set `min-camera-orbit` / `max-camera-orbit`. If you'd like us to auto-frame for you, you can also add some unreferenced vertices (or other tiny geometry) that contain the space your animation will encompass and then we'll fit to that.

**Reply by bigrig2212** [reply]
Aha! Brilliant. Got it. Thx.
